libs/env/src/env.ts

```diff
@@ -554,7 +554,7 @@ const envDefinitions = {
     desc: "SimpleHash API base url",
   },
   SKIP_ONBOARDING: {
-    def: false,
```

Comment:
the default shouldn't be true.

Reply:
* (ok, i didn't saw the latest commit. but just know that since it's an open PR where i'm pinged on, i naturally reviewed it)